docs/community/dev-card.md

```diff
@@ -0,0 +1,16 @@
+---
+id: dev-card
+title: 'Showcase your open source stats with your dev card'
+sidebar_label: 'Dev Card'
+keywords:
+- dev-card 
+---
+
+The Dev card feature is what we like to call your "open source business card". At the front, you see your username, the amount of pull requests you create, repositories you contribute to, a `View Profile` button that takes you to your dashboard, and a green icon, which describes your activity rate.
+
+When you click on the back of your card, you see the same information that's shown in the front as well as the headline from your OpenSauced profile, amount of time you've been using our product, and your [PR velocity rate](../../docs/community/pr-velocity.md).
```

Comment:
Let's change "you see the same information" to "you see additional information, including ..."

I believe that the date isn't when you started using the product. I think it's your first commit. @bdougie can you verify that?

Reply:
Ok, that date is the first contribution date on github